doc/sphinxext/numpydoc/docscrape_sphinx.py

```diff
@@ -33,6 +33,7 @@ def load_config(self, config):
         self.use_plots = config.get('use_plots', False)
         self.use_blockquotes = config.get('use_blockquotes', False)
         self.class_members_toctree = config.get('class_members_toctree', True)
+        self.attributes_as_param_list = config.get('attributes_as_param_list', True)
```

Comment:
Will this ensure that Attributes is formatted like a table?

Reply:
Yes (updated the top post). For now, that is the only change I made to the numpydoc source (it's in a separate commit).

But, if I am the only one that finds that better, I am OK to leave that.

Reply:
Well, there isn't ;-) That's something I added here (but with only a few lines: https://github.com/pandas-dev/pandas/pull/20383/commits/075438f4b5ba14df925e79f00854c2d4b61e0cae), and that's what I proposed to do in https://github.com/numpy/numpydoc/pull/106, so it would be nice to have this in upstream)